lms/services/group_info.py

```diff
@@ -61,3 +67,5 @@ def upsert(self, authority_provided_id, consumer_key, params):
             group_info.upsert_instructor(
                 dict(email=self._lti_user.email, **self._lti_user.h_user._asdict())
             )
+
+        group_info.info["type"] = type_
```

Comment:
This little guy is a bit mysterious on his own. As we interact with instructors above using a method (which internally uses a property over `info`) should this be a property too? Otherwise you have to jump between the two files to see how `info` is built up.

Reply:
Done c24dc94e6ab1768ee52ca34e11ed91eb42713f37